src/serial.rs

```diff
@@ -58,11 +58,27 @@ pub enum Event {
     Txe,
     /// Idle line state detected
     Idle,
+
+    ///Tx threshlold interrupt enable
+    Txftie,
+
+    ///Rx threshlold interrupt enable
+    Rxftie,
 }
 
 pub mod config {
     use crate::time::Hertz;
 
+    #[derive(Copy, Clone, PartialEq)]
+    pub enum FifoThreshold {
+        OneEighth,
+        OneQuarter,
+        OneHalf,
```

Comment:
For consistency with the DMA API, it's nicer to just say "Quarter", "Half"...

```suggestion
        Eighth,
        Quarter,
        Half,
```

Reply:
Ok thanks